app/controllers/inventories_controller.rb

```diff
@@ -0,0 +1,58 @@
+class InventoriesController < ApplicationController
+  before_action :authenticate_user!
+
+  def index
+    @user = current_user
+    @inventories = @user.inventory
```

Comment:
[OPTIONAL]
I think you can simplify like this:

`@inventories = current_user.inventory` and in your views you can access the current user with `inventory.user`.

The same with the other parts where you use it.

Reply:
@paulsaenzsucre, You are right. But I do not need all users for all inventory because all inventories are from the same user. Add in index.erb.html we will have an array of inventories. So if all inventories include users it will be more costly than the current version as per my thought.